lib/caesar.rb

```diff
@@ -1,6 +1,58 @@
+require 'pry'
 class Caesar
   attr_reader :key
 
   def initialize
+    @key = 3
+    @alphabet =* "a".."z"
+    @cap_alphabet =* "A".."Z"
   end
-end
\ No newline at end of file
+
+  def self.load_text(file)
+    text = File.open(file, "r")
+    full_text = []
+    text.each_line do |line|
+      full_text << line.to_s
+    end
+    # binding.pry
+    full_text = full_text.join
+    full_text.gsub("\n","")
+  end
+
```

Comment:
The spec says there should be a method from_file that reads the file and translates it

Reply:
I’m sorry, I should have paid more attention to the spec, I didn’t see that the load method needed to have a specific name.